jep/1/README.adoc

```diff
@@ -0,0 +1,536 @@
+themselves has decided that the JEP is actually a bad idea, or has
+accepted that a competing proposal is a better alternative.
+
+When a JEP is Accepted, Rejected or Withdrawn, the JEP should be updated
+accordingly. In addition to updating the status field, at the very least
+the Resolution header should be added with a link to the relevant post
+in the python-dev mailing list archives.
+
+JEPs can also be superseded by a different JEP, rendering the original
+obsolete. This is intended for Informational JEPs, where version 2 of
+an API can replace version 1.
+
+The possible paths of the status of JEPs are as follows:
+
+image::workflow.png[JEP Workflow]
+
+Some Informational and Process JEPs may also have a status of "Active" if they
+are never meant to be completed. E.g. JEP 1 (this JEP).
+
+
+=== JEP Maintenance
+
+In general, Standards track JEPs are no longer modified after they have
+reached the Final state. Once a JEP has been completed, Jenkins developer
+documentation must become the formal documentation of the expected behavior.
+
+Informational and Process JEPs may be updated over time to reflect changes
+to development practices and other details. The precise process followed in
+these cases will depend on the nature and purpose of the JEP being updated.
+
+
+== What belongs in a successful JEP?
+
+Each JEP should have the following parts:
+
+. **Metadata** - table containing metadata about the JEP, including the JEP
+  number, a short descriptive title, the names, and optionally the contact info
+  for each author, etc.
+. **Abstract** - short (200 word) description of the technical issue
+  being addressed.
+. **Specification** - The technical specification should describe the
+  syntax and semantics of any new feature. The specification should be
+  sufficiently detailed to allow new, or existing, Jenkins developers to
+  reasonably understand the scope/impact of an implementation.
+. **Motivation** - The motivation is critical for JEPs that want to
+  change Jenkins itself. It should clearly explain why the
+  existing code base is inadequate to address the
+  problem that the JEP solves. JEP submissions without sufficient
+  motivation may be rejected outright.
+. **Rationale** - The rationale fleshes out the specification by
+  describing what motivated the design and why particular design
+  decisions were made. It should describe alternate designs that
+  were considered and related work, e.g. how the feature is supported
+  in other languages.
++
+The rationale should provide evidence of consensus within the
+community and discuss important objections or concerns raised
+during discussion.
+
+. **Backwards Compatibility** - All JEPs that introduce backwards
+  incompatibilities must include a section describing these
+  incompatibilities and their severity. The JEP must explain how the
+  author proposes to deal with these incompatibilities. JEP
+  submissions without a sufficient backwards compatibility treatise
```

Comment:
I think "treatise" is too strong a word here; it's sort of like "manifesto." I would say, "JEP submissions that do not adequately discuss backwards compatibility, when such discussion is required, may be rejected outright."